### DISCUSSION TITLE
Table of contents / outline feedback

### DISCUSSION THREAD

**Discussion by dmadisetti** [marimo-team/marimo#1682] [General]
Just some quality of life suggestions for markdown headings.

 - Currently only to h3 is supported (https://github.com/marimo-team/marimo/issues/1617), maybe more would be considered if collapsable headings were enabled in the TOC?

 - Another issue suggests collapsable blocks by heading: https://github.com/marimo-team/marimo/issues/1085
Super useful for large notebooks

 - Colab has a "run section" button in their TOC, which is also nice

**Comment by akshayka** [comment]
@dmadisetti I also often use h4's; down to add support for that at least.

Collapsible headers would be awesome.
